lib/index.js

```diff
@@ -572,12 +587,29 @@ internals.getJoinTableInfo = (model, knex, cb) => {
                         skippedCols.push({
                             tableName,
                             column,
-                            type
+                            type,
+                            schemaConflict: false // Set as a default, may be overridden below
```

Comment:
Setting a default here to be overridden later is meh, idk doesn't feel great but thoughts?